odoo/orm/fields_textual.py

```diff
@@ -707,12 +712,16 @@ def get(self, key, default=None):
             return default
         if vals is None:
             return None
+        if not (self._field.compute or (self._field.store and (key or key.origin))):
+            return vals.get(self._lang, vals.get('en_US', default))
         return vals.get(self._lang, default)
 
     def __getitem__(self, key):
         vals = self._cache[key]
         if vals is None:
             return None
+        if not (self._field.compute or (self._field.store and (key or key.origin))):
+            return vals.get(self._lang, vals.get('en_US'))
```

Comment:
```suggestion
            return vals.get(self._lang, vals['en_US'])
```
Technically, I expect a `MissingError` if `en_US` is not available, which also means we have a coding error somewhere else. But maybe it would be safer to use `get` in stable?